test_runner/src/main/kotlin/ftl/args/ValidateCommonArgs.kt

```diff
@@ -8,13 +8,22 @@ fun CommonArgs.validate() {
     assertShardTime()
     assertRepeatTests()
     assertSmartFlankGcsPath()
+    assertOrientationCorrectness()
+}
+
+private fun CommonArgs.assertOrientationCorrectness() {
+    val orientationsList = listOf("portrait", "landscape", "default")
+    devices.map { it.orientation }.forEach { ori ->
+        if (orientationsList.filter { ori.toLowerCase().contentEquals(it) }.size != 1)
```

Comment:
Maybe
```
      if (ori.toLowerCase() !in orientationsList)

```
We can configure more thane one device so
what do you think about display all misspelled orientations on fail?

Reply:
well i must say that `!in `is a great keyword i am not familiar with thanks.
Totally doable :)